src/views/checkout/AddressInformationView.vue

```diff
@@ -112,3 +112,4 @@ function updateBillingAddressOfOrderInformation() {
     order.value.billingAddress = selectedBillingAddress.value
 }
 </script>
+@/model/classes/addressImpl
```

Comment:
what's this?

Reply:
Oh dear, when refactoring, some things got out of hand somehow. It seems like I didn't clean up properly. Will fix that.

Reply:
0d5690853915dfb489187665c42f8e89fc45a131